spacelift/data_stack.go

```diff
@@ -154,6 +154,20 @@ func dataStack() *schema.Resource {
 				Description: "Name of the Docker image used to process Runs",
 				Computed:    true,
 			},
+			"showcases": {
```

Comment:
Why plural?

Reply:
Because we may have multiple showcases in the future? But you're right, maybe it should still read Showcase.

Reply:
Still, that's just a block name.